pyvistaqt/plotting.py

```diff
@@ -298,6 +298,18 @@ def render(self) -> None:
         """Override the ``render`` method to handle threading issues."""
         return self.render_signal.emit()
 
+    @wraps(BasePlotter.enable)
+    def enable(self) -> None:
+        """Wrap ``BasePlotter.enable``."""
+        self.interactor.setEnabled(True)
```

Comment:
don't use `self.interactor` anymore, it's just `self`

Reply:
Ah good catch! It had been a while since I've been around this source code